packaging/packager_rpm.sh

```diff
@@ -60,6 +63,7 @@ sleep 3  # give xvfb some time to start
 
 # unit tests
 dnf install -y python3-PyQt5
+pip3 install PyQt5
```

Comment:
Same here as above

Reply:
This duplicated imports can be cutted out, I'll change them